transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -390,6 +390,8 @@ protected void run() {
                     //increase the size of the array as we needed the whole space for the events
                     events.increase();
                 }
+            } catch (ThreadDeath td) {
+                throw (ThreadDeath) td;
```

Comment:
Right below these catch clauses it says "Always handle shutdown even if the loop processing threw an exception."

Should that perhaps be moved into a finally clause? It's the same deal in all of the event loops modified in this PR.

Reply:
good idea @chrisvest ... @greenjustin can you please adjust ?

Reply:
Sure, how does this look?